modular_skyrat/modules/gunsgalore/code/guns/akm/akm.dm

```diff
@@ -42,7 +42,7 @@
 	multiple_sprites = AMMO_BOX_FULL_EMPTY
 
 /obj/item/gun/ballistic/automatic/assault_rifle/akm/modern
-	name = "\improper Tactical AKMS"
+	name = "\improper tactical AKMS"
```

Comment:
Does it still need a `\improper` here? I actually don't remember how that works.

Reply:
Hmmm I'll have to refresh myself on `\improper` and come back to this.